packages/benchmarks/src/benchmarks/blinkdb.ts

```diff
@@ -0,0 +1,87 @@
+import { randFirstName } from "@ngneat/falso";
+import { clear, createDB, createTable, first, insert, remove, update } from "blinkdb";
+import { compare } from "../framework";
+
+interface User {
+  id: number;
+  name: string;
+  age?: number;
+}
+
+(async () => {
+  // BlinkDB setup
+  const blinkdb = createDB({
+    clone: true,
```

Comment:
I've set `clone` to true in this file because otherwise the "updating one item" benchmark doesn't work correctly (because the same user is present in the table with and without index).

Since I just used the benchmark to compare blinkDB with itself (so that I know whether my changes are actually helpful), I thought that `clone: true` should be fine as well (especially since it's the default option and will therefore be used the most).

However, if you want to, then I can change it to `false` and adapt the benchmarks so that the two tables have different user objects instead of using the same

Reply:
Ah, that makes sense. In that case, you can remove `clone: true` altogether, since it's the default with `createDB()`.